app/command_loadtest.go

```diff
@@ -194,12 +206,16 @@ func (me *LoadTestProvider) SetupCommand(a *App, args *model.CommandArgs, messag
 		}
 	}
 	client := model.NewAPIv4Client(args.SiteURL)
+	client.SetToken(args.Session.Token)
```

Comment:
This was the primary source of the problem: no token was provided for the initial `CreateBasicUser` call on line 212 below.